### PR TITLE
Update in.m3u

### PR DESCRIPTION
**Pull request by desiflex**

### CODE REVIEW

**Comment by desiflex** [comment]
UPDATE PLS

File: channels/in.m3u [hunk 1]
```diff
@@ -9,8 +9,8 @@ https://6n3yogbnd9ok-hls-live.5centscdn.com/threetamil/d0dbe915091d400bd8ee7f27f
 http://103.199.161.254/Content/7smusic/Live/Channel(7smusic)/index.m3u8
 #EXTINF:-1 tvg-id="9XM.in" tvg-name="9XM" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/fD7wLka.jpg" group-title="Music",9XM (480p)
 https://d2q8p4pe5spbak.cloudfront.net/bpk-tv/9XM/9XM.isml/index.m3u8
-#EXTINF:-1 tvg-id="ANEWSJATTTVBhangraFlava.in" tvg-name="A NEWSJATT TV- Bhangra Flava" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Music",A NEWSJATT TV- Bhangra Flava (720p)
-http://newsjatt.camdvr.org:1935/newsjatt/myStream/playlist.m3u8
```
Comment (themak1985) [inline channels/in.m3u:13]: This is still valid.

File: channels/in.m3u [hunk 2]
```diff
@@ -9,8 +9,8 @@ https://6n3yogbnd9ok-hls-live.5centscdn.com/threetamil/d0dbe915091d400bd8ee7f27f
...
+#EXTINF:-1 tvg-id="ANEWSDONSTVBhangraFlava.in" tvg-name="A NEWSDONS TV- Bhangra Flava" tvg-country="IN" tvg-language="Punjabi" tvg-logo="" group-title="Music",A NEWSDONS TV- Bhangra Flava (720p)
+http://tiny.cc/jattvlink
```
Comment (themak1985) [inline channels/in.m3u:13]: This looks like a totally different stream. I would rather you add as a new stream than replace an existing.

**Review by themak1985 — CHANGES_REQUESTED** [review]

File: channels/in.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/in.m3u:13]: The funny thing is that why does it need to go through a tiny.cc link when you simply can give the stream link ?

**Comment by Sphinxroot** [comment]
Double PR #2999

**Pull request closed without merging**